### PR TITLE
Activating git.legal

### LINKED ISSUE #840: Activate git.legal
## Description
We need to check the compliance of our licenses. Activate the [git.legal](https://docs.codeclimate.com/docs/git-legal) plug-in in the [CodeClimate.yml file](https://github.com/raiden-network/light-client/blob/master/.codeclimate.yml).

## Acceptance criteria
- 

## Tasks
- [ ]

### PR DESCRIPTION
**Pull request by taleldayekh**
Closes #840

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -12,6 +12,6 @@ checks:
   similar-code:
     config:
       threshold: 100
-      plugins:
-        git-legal:
-          enabled: true
+plugins:
+    git-legal:
+      enabled: true
```
Comment (kelsos) [inline .codeclimate.yml]: Can we also make sure that we don't allow copyleft? gnu public license and affero?

https://docs.codeclimate.com/docs/git-legal#section-configure-license-policy

Comment (taleldayekh) [reply .codeclimate.yml]: Of course, I will add that.

Comment (kelsos) [reply .codeclimate.yml]: I am sorry but I don't see these following lines in the PR
```yaml
   config:
      allow_affero_copyleft: false
      allow_strong_copyleft: falsed
```

Comment (taleldayekh) [reply .codeclimate.yml]: They're there. View the file and you'll see the changes. :)

Comment (kelsos) [reply .codeclimate.yml]: Damn, I was checking only the last commit apparently :'(

**Pull request merged**